se.redfield.bert/src/se/redfield/bert/prefs/CustomCondaEnvironmentsPreferencePanel.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2021 Redfield AB.
```

Comment:
what license? 2022?

Reply:
I just kept with the project settings here.